Makefile

```diff
@@ -211,13 +211,18 @@ local-unit-test-cxg-admin:
 #     | jq -r '"AWS_ACCESS_KEY_ID=\(.Credentials.AccessKeyId) AWS_SECRET_ACCESS_KEY=\(.Credentials.SecretAccessKey) AWS_SESSION_TOKEN=\(.Credentials.SessionToken)"') \
 # 	AWS_REGION=us-west-2 BOTO_ENDPOINT_URL= DEPLOYMENT_STAGE=staging make local-functional-test)
 .PHONY: local-functional-test
-local-functional-test: export AWS_PROFILE=$(TEST_AWS_PROFILE)
 local-functional-test: ## Run functional tests in the dev environment
 	if [ -n "$${BOTO_ENDPOINT_URL+set}" ]; then \
 		EXTRA_ARGS="-e BOTO_ENDPOINT_URL"; \
 	fi; \
 	chamber -b secretsmanager exec corpora/backend/$${DEPLOYMENT_STAGE}/auth0-secret -- \
-		docker-compose $(COMPOSE_OPTS) run --rm -T -e CLIENT_ID -e CLIENT_SECRET -e TEST_ACCOUNT_USERNAME -e TEST_ACCOUNT_PASSWORD -e DEPLOYMENT_STAGE -e AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY -e AWS_SESSION_TOKEN $${EXTRA_ARGS} \
+		docker-compose $(COMPOSE_OPTS) run --rm -T \
+		-e CLIENT_ID -e CLIENT_SECRET \
+		-e FUNCTEST_ACCOUNT_USERNAME -e FUNCTEST_ACCOUNT_PASSWORD \
+		-e TEST_AUTH0_USER_ACCOUNT_PASSWORD -e TEST_APP_ID -e TEST_APP_SECRET \
```

Comment:
...while this would be for getting past the outer auth0 layer / proxy, right?

Reply:
My resulting question then would be: why wasn't the functest_account_username secret being used before now...

Reply:
> ...while this would be for getting past the outer auth0 layer / proxy, right?

yes for `-e TEST_APP_ID -e TEST_APP_SECRET`

Reply:
> y resulting question then would be: why wasn't the functest_account_username secret being used before now...

So they actually were being used, but they were getting introduced into the container through a different path. In the [GHA functional test](https://github.com/chanzuckerberg/single-cell-data-portal/blob/main/.github/workflows/deploy-happy-stack.yml.), the container was getting the secret directly from AWS Secrets manager using credentials supplied to the docker container. The AWS credentials were passed into the GHA container here https://github.com/chanzuckerberg/single-cell-data-portal/blob/3bca29bd6cef0b8d0eab6fdd49edf57cd8741364/Makefile#L220
but this didn't work when running the tests locally or when `DEPLOYMENT_STAGE=test` which is why this comment
https://github.com/chanzuckerberg/single-cell-data-portal/blob/3bca29bd6cef0b8d0eab6fdd49edf57cd8741364/Makefile#L206-L212

Now all of the secrets needed by the functional tests are introduced via the environment using the make recipe and chamber. No more sneaking secret getting introduced from multiple different sources. The functional tests no longer need to access AWS secrets to work.

Reply:
I'm going to remove the AWS credentials and the comment from the make recipe. They aren't needed.

Reply:
excellent, thanks!